appveyor.yml

```diff
@@ -4,8 +4,8 @@
 os: Unstable
 version: 0.3.{build}.0
 install:
-  - cinst -y pester
   - ps: Get-PackageProvider -Name NuGet -Force

```

Comment:
I think we should do the same two calls as in the DscResource.Tests? It was suggested by PowerShell Gallery team. https://github.com/PowerShell/DscResource.Tests/blob/a522ade40d959fc931dfc9eb2a4ec096b7e3afb1/AppVeyor.psm1#L46-L49